extensions/parquet/table/src/main/java/io/deephaven/parquet/table/location/MinMax.java

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright (c) 2016-2025 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.parquet.table.location;
+
+import io.deephaven.annotations.CopyableStyle;
+import io.deephaven.util.compare.ObjectComparisons;
+import org.immutables.value.Value.Check;
+import org.immutables.value.Value.Immutable;
+import javax.annotation.Nullable;
+
+@Immutable
+@CopyableStyle
+abstract class MinMax {
+
+    @Nullable
+    abstract Object min();
+
+    @Nullable
+    abstract Object max();
```

Comment:
I'm comparing this interface to `org.apache.parquet.column.statistics.Statistics`, possibly we want to mimic Statistics more closely.